doc/sphinx/source/examples.rst

```diff
@@ -417,15 +399,13 @@ for mass and energy densities. This problem can be run with::
    ./navierstokes -problem density_current
 
 
-.. _BPs:
-
-Benchmark Problems
---------------------------------------
+Bakeoff Problems
+================
 
-The Center for Efficient Exascale Discretizations (CEED) uses Benchmark Problems (BPs)
+The Center for Efficient Exascale Discretizations (CEED) uses Bakeoff Problems (BPs)
 to test and compare the performance of high-order finite element implementations. The
 definitions of the problems are given on the ceed
-`website <https://ceed.exascaleproject.org/bps/>`_. Each of the following benchmark
+`website <https://ceed.exascaleproject.org/bps/>`_. Each of the following bakeoff
```

Comment:
Thanks. I thought that since in the examples we use the whole T-vector to T-vector steps (with the parallel scatter/gather), it was more appropriate to call them Benchmark problems, rather than Bakeoff problems as it says on the website.

Reply:
I guess I don't follow. What we do seems to match the spec (when run in parallel).
> These are all T-vector-to-T-vector and include parallel scatter + element scatter + element evaluation kernel + element gather + parallel gather.
https://ceed.exascaleproject.org/bps/